react-hot-button-component/my-app/src/CustomButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+type Props = {
+  text: string;
+};
+
+export default function CustomButton({ text }: Props) {
+  let [index, setClicked] = useState(0);
```

Comment:
Don't you get a lint warning that this should be const? (It should be, by the way.)

Reply:
![Screenshot 2023-05-22 at 10 00 09 AM](https://github.com/DylanPDao/c0323-code-solutions/assets/20610202/30021bc8-4ad8-49fa-bcc5-fd3ce083940d)

Reply:
No lint for some reason.